packages/api/tests/endpoints/test-rules.js

```diff
@@ -729,14 +734,16 @@ test('PUT replaces a rule', async (t) => {
       provider: undefined,
     }
   );
+  const translatedPgRecord = await translatePostgresRuleToApiRule(originalPgRecord, testKnex);
 
   const updateRule = {
-    ...omit(originalDynamoRule, ['queueUrl', 'provider', 'collection']),
+    ...omit(translatedPgRecord, ['queueUrl', 'provider', 'collection']),
     state: 'ENABLED',
     // these timestamps should not get used
     createdAt: Date.now(),
     updatedAt: Date.now(),
   };
+  console.log('UPDATE RULE', updateRule);
```

Comment:
Remove this

Reply:
[`4c12492` (#2871)](https://github.com/nasa/cumulus/pull/2871/commits/4c1249251939ede5e8b8d6fe90b051ca72effc6d)